addons/stock/views/stock_config_settings_views.xml

```diff
@@ -7,88 +7,345 @@
+                                    <label for="module_delivery_temando"/>
+                                    <div class="text-muted">
+                                        Ship with Temando
+                                    </div>
+                                    <div class="content-group">
+                                        <div class="text-warning mt16" id="delivery_temando" attrs="{'invisible': [('module_delivery_temando','=',False)]}">
+                                            Save this page and come back here to set up the feature.
+                                        </div>
+                                    </div>
+                                </div>
+                            </div>
+                        </div>
+                        <h2>Products</h2>
+                        <div class="row mt16 o_settings_container">
+                            <div class="col-md-6 o_setting_box">
+                                <div class="o_setting_left_pane">
+                                    <field name="group_product_variant"/>
+                                </div>
+                                <div class="o_setting_right_pane">
+                                    <label for="group_product_variant"/>
+                                    <div class="text-muted">
+                                        Set product attributes (e.g. color, size) to manage variants
+                                    </div>
+                                    <div class="content-group">
+                                        <div class="text-warning mt16" attrs="{'invisible': [('group_product_variant', '=', False)]}">
+                                            <button name="%(product.attribute_action)d" icon="fa-arrow-right" type="action" string="Attributes" class="btn-link" style="padding-left: 0;"/>
+                                        </div>
+                                    </div>
+                                </div>
+                            </div>
+                            <div class="col-md-6 o_setting_box">
+                                <div class="o_setting_left_pane">
+                                    <field name="group_uom"/>
+                                </div>
+                                <div class="o_setting_right_pane">
+                                    <label for="group_uom"/>
+                                    <div class="text-muted">
+                                        Sell and purchase products in different units of measure
+                                    </div>
+                                    <div class="content-group">
+                                        <div class="text-warning mt16" attrs="{'invisible': [('group_uom', '=', False)]}">
+                                            <button name="%(product.product_uom_form_action)d" icon="fa-arrow-right" type="action" string="Units Of Measure" class="oe_link" style="padding-left: 0;"/>
```

Comment:
Replace class="oe_link" with class="btn-link" and remove style="padding-left: 0;" .